grails-app/controllers/org/pih/warehouse/api/ReplenishmentApiController.groovy

```diff
@@ -117,10 +111,6 @@ class ReplenishmentApiController {
             replenishment.replenishmentNumber = grailsApplication.config.openboxes.stockTransfer.binReplenishment.prefix + identifierService.generateOrderIdentifier()
         }
 
-        if (jsonObject.status) {
-            replenishment.status = ReplenishmentStatus.valueOf(jsonObject.status)
-        }
-
```

Comment:
out of curiosity - what problem did it cause? Is it binded correctly without that?